src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -453,4 +453,31 @@ object DepositHandler {
 
     getFile(bagStoreSettings.baseDir, topDir, uuidDir, version)
   }
+
+  def writeToProperties(bagDir: File)(implicit settings: Settings, id: String): Try[Unit] = Try {
+    for {
+      bag <- getBag(bagDir)
+      resources <- getResources(bag)
```

Comment:
As agreed in our discussion, I left this code in DepositHandler.  There is a check in StatementManagerImpl that takes care that the client gets to see the file uris only when the state is ARCHIVED